projects/js-packages/social-logos/.gitattributes

```
@@ -0,0 +1,14 @@
+# Files not needed to be distributed in the package.
+.gitattributes    export-ignore
+node_modules      export-ignore
+
+# Files to include in the mirror repo, but excluded via gitignore
+# Remember to end all directories with `/**` to properly tag every file.
+/build/**         production-include
+
+# Files to exclude from the mirror repo
+/changelog/**     production-exclude
+/.eslintrc.cjs    production-exclude
+/jest.config.cjs  production-exclude
+**/stories/**     production-exclude
+/tests/**         production-exclude
```

A few changes in here:
* No need to publish `src/` or `tools/`, we only need the built files.
* No need to publish `.eslintignore`.
* We never actually use the `build/svg-clean` files in the package, probably no reason to publish them.

```suggestion
/build/svg-clean/**     production-exclude
/changelog/**     production-exclude
/.eslintignore    production-exclude
/.eslintrc.cjs    production-exclude
/jest.config.cjs  production-exclude
**/stories/**     production-exclude
/src/**           production-exclude
/tests/**         production-exclude
/tools/**         production-exclude
```